src/components/Markdown.svelte

```diff
@@ -0,0 +1,26 @@
+<script>
```

Comment:
(We use the same [Markdown component](https://github.com/mozilla/glean-dictionary/blob/main/src/components/Markdown.svelte) in Glean Dictionary.)